pkg-example-amf-bp/amfdeployment.yaml

```diff
@@ -1,5 +1,7 @@
 apiVersion: workload.nephio.org/v1alpha1
-kind: AMFDeployment
+kind: NFDeployment
 metadata:
   name: amf-example
   namespace: amf-example
+spec:
+  provider: amf.free5gc.io
```

Comment:
I think that your editor has added a Line Ending, this could be happened if you are using a Windows editor. Maybe using config in Notepad++ can avoid it.